### DISCUSSION TITLE
'Permanent' Queue, or Automatically add queued songs to default playlist

### DISCUSSION THREAD

**Discussion by Svindle** [jagrosh/MusicBot#1406] [Ideas]
Migrating from previous music bots, my discord could use a feature along these lines. Any played song would be added to a sort of 'default' playlist (by anyone with the DJ role) that would persist through restarts. The repeat flag for the queue is close to this, but everything is lost when the bot is restarted.

**Comment by PokiCo1** [comment]
Would love this
function to move from queue to a playlist
